docs/5.x/distributing-your-plugin.md

```diff
@@ -42,6 +42,8 @@ To make your plugin shine on the Piwik marketplace, include screenshots in your
 Prepare a few screenshots of your plugin in action and place them in a `screenshots/` directory in your plugin's folder.
 Give them descriptive names because the file names will be used as the legends shown below each screenshot. Only alphanumeric characters, underscores and dashes are allowed in the file name. The file name must end with `.png`, `.jpg` or `.jpeg`.
 
+To add a cover image, it must be 880x480 pixels and named `_cover.png`. It can then be added to the `screenshots/` directory with the rest of the plugin screenshots. If the cover image does not fit the recommended dimensions, it may not display correctly.
```

Comment:
@snake14 looks good, should we make 880x480 as bold ?
```suggestion
To add a cover image, it must be **880x480** pixels and named `_cover.png`. It can then be added to the `screenshots/` directory with the rest of the plugin screenshots. If the cover image does not fit the recommended dimensions, it may not display correctly.
```

Reply:
Thank you for the good suggestion @AltamashShaikh